import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java

```diff
@@ -737,20 +737,24 @@ protected void instantiateStaticFields() {
 		Field participatingOrg = new Field("Funding Organization", "participating-org",
                 FieldType.ORGANIZATION,true, getTranslationForField("participating-org"));
 		participatingOrg.setSubType("Funding");
+		participatingOrg.setSubTypeCode("1");
```

Comment:
lets try to use constants so its more clear to know that subtype 1 is for participating-org